Update change-plans-manually.md

jofried opened the pull request:
adding note to clarify that when you are renewing an office 365 subscription, if the renewal is changing the sku to a Microsoft 365 sku, the administrator will need to update the license assignment to reflect this sku change

jofried committed:
Update change-plans-manually.md

adding note to clarify that when you are renewing an office 365 subscription, if the renewal is changing the sku to a Microsoft 365 sku, the administrator will need to update the license assignment to reflect this sku change

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

JohanFreelancer9 commented on microsoft-365/commerce/subscriptions/change-plans-manually.md:
```diff
@@ -38,6 +38,14 @@ To continue with a manual change, read [Step 2: Buy a new subscription](#step-2-
 > [!IMPORTANT]
 > If you are changing to a plan with fewer data-related services than your current plan (downgrading), you need to manually back up any data you wish to keep. For more information, see [Back up data before changing plans](back-up-data-before-switching-plans.md).
 
+>[!NOTE]
+>If you are renewing your Office 365 subscription and are upgrading from an Office 365 sku to a Microsoft 365 sku, this will require you to change the license assignment for all users. The switch from an Office 365 sku to a Microsoft 365 sku will not happen automatically.
+>
+>For Example:
+>
+>If users are currently assigned "Office 365 E3", and you renew your subscription to "Microsoft 365 E3", you will need to update the user license assignments to reflect this sku change.
+
```
```suggestion
> [!NOTE]
> If you are renewing your Office 365 subscription and are upgrading from an Office 365 licensing plan (SKU) to a Microsoft 365 SKU, this will require you to change the license assignment for all users. The switch from an Office 365 SKU to a Microsoft 365 SKU will not happen automatically. For example, If users are currently assigned an Office 365 E3 SKU and you renew your subscription to Microsoft 365 E3, you will need to update the user license assignments to reflect this SKU change.
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @jofried, please commit my suggested changes if you agree with them. Many thanks.

jofried committed:
Update microsoft-365/commerce/subscriptions/change-plans-manually.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

yogkumgit commented:
@cmcatee-MSFT Please merge this PR. Thanks

cmcatee-MSFT commented:
@yogkumgit Did this change stem from customer feedback? If so, please provide a link to it so I can review. Thanks.

yogkumgit commented:
@cmcatee-MSFT No this is not from customer feedback. Thanks

cmcatee-MSFT committed:
Update change-plans-manually.md

Moved proposed note to Step 4: Reassign the licenses, and edited the paragraph for technical accuracy.
